templates/entity/create/create-common.jade

```diff
@@ -0,0 +1,24 @@
+extends /layout
+
+block body
+    .container#content
+        .row
+            h1=heading
+                small=' ' + subheading
+        .row
+            noscript
+                .alert.alert-danger(role='alert') This page will not function correctly without JavaScript! Please enable JavaScript to use this page.
+        if script == 'creator'
```

Comment:
Alternatively, to get rid of these if/else statements, change the ID of these form elements to "#entityForm", then update the corresponding `controllers/entity/` scripts to use this element.